Core/Inc/ACDC_SPI.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file ACDC_SPI.h
+ * @author Liam Bramley
+ * @brief Header file for SPI driver
+ * @version 0.1
+ * @date 2024-01-30
+ * @copyright Copyright (c) 2023-2024
+ * 
+ */
+
+#ifndef ACDC_SPI_H
+#define ACDC_SPI_H
+
+#include "stm32f1xx.h"
+#include "ACDC_GPIO.h"
+
+/// @brief Initializes the SPI given
+/// @param SPI SPI to initialize
+void SPI_Initalize(const SPI_TypeDef *SPI);
+
+/// @brief Transmits data over the given SPI
+/// @param data Data to transmit
+/// @param SPI SPI to transmit over
+void SPI_Transmit(uint16_t data, const SPI_TypeDef *SPI);
```

Comment:
For consistency and to keep it standardized the SPI_TypeDef should be the first parameter. What do you think @joespexarth

Reply:
I'd agree, keeps everything uniform.